addon/mixins/base-serializer.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+import partialModelHandler from 'ember-data-partial-model/utils/serializer-partial-model-handler';
+const { Mixin } = Ember;
+const { preNormalize, postSerialize } = partialModelHandler;
+
+export default Mixin.create({
+  normalize: function(modelClass, resourceHash) {
+    preNormalize(modelClass, resourceHash, this, this._normalizePartialRelationship);
+    return this._super(modelClass, resourceHash);
+  },
+
+  serialize: function(snapshot /*, options */) {
+    let serializedHash = this._super(...arguments);
+    postSerialize(snapshot, serializedHash, this._copyAttributesFromPartialToParent,
+      this._modifyHashBasedOnDescriptorKey);
+
+    return serializedHash;
+  },
+
+  serializeBelongsTo: function(/*snapshot, json, relationship */) {
```

Comment:
You will break everyone's relationship serializing by doing this. Need to comment out.